src/main/java/core/basesyntax/services/TransactionService.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.services;
+
+import core.basesyntax.FruitTransaction;
+import java.util.List;
+
+public interface TransactionService {
```

Comment:
Rename this interface to ParserService with method parse, and class implementation also to ParserServiceImpl